libdec/printer.js

```diff
@@ -16,6 +16,37 @@
  */
 
 (function() {
+    function initializeColors() {
+        const useR2Colors = true; // hardcoded. TODO. add a config var
+        const config = {
+            "callname": "gray",
+            "integers": "cyan",
+            "comment": "red",
+            "labels": "green",
+            "types": "green",
+            "macro": "yellow",
+            "flow": "magenta",
+            "text": "yellow"
+        };
+        if (useR2Colors) {
```

Comment:
Change this into `if (r2cmd) {`

Reply:
what about having a config option to pick that? shouldnt r2cmd be always available?

Reply:
not on the testsuite